contentpacks/khanacademy.py

```diff
@@ -894,6 +895,14 @@ def retrieve_assessment_item_data(assessment_item, lang=None, force=False, no_it
     with open(path, "r") as f:
         item_data = json.load(f)
 
+    # TEMP HACK: translate the item text here before URLs are localized, because otherwise, later, Crowdin strings no longer match
+    if lang != "en" and content_catalog is not None:
```

Comment:
The `content_catalog` and `translate_assessment_item_text` should be reference first else we will have `Unresolved reference` error.